flex advance mode columns to use available space

AlanBell opened the pull request:
as discussed in #22009 this allows columns in advance mode to be a bit wider to fill the screen, they have a min-width of 350 so if there isn't enough screen they scroll as before

AlanBell committed:
flex advance mode columns to use available space

ineffyble commented:
Alternative to #20225.

ineffyble commented:
I suspect this is going to look weird on ultrawide aspect ratios, will test on my monitor tonight.

AlanBell committed:
Update components.scss

limit the flexing of the drawer and columns, allow columns to go slightly thinner before scrolling

AlanBell commented:
I put some max-width limits to avoid it looking weird on wide viewports with few columns.

mjankowski commented:
Closing due to sheer age and lack of rebase -- and separately, I believe a similar PR which adjusted the widths of advanced mode columns merged recently (will try to dig up).

That said, if this is still relevant, can you rebase/reopen?
